src/core/PlayArea.ttslua

```diff
@@ -3,14 +3,17 @@
 ---------------------------------------------------------
 
 -- set true to enable debug logging
-local DEBUG = false
+local DEBUG = true
 
 -- Location connection directional options
 local BIDIRECTIONAL = 0
 local ONE_WAY = 1
+local INCOMING_ONE_WAY = 2
 
 -- Connector draw parameters
 local CONNECTION_THICKNESS = 0.015
+local DRAGGING_CONNECTION_THICKNESS = 0.15
```

Comment:
Why do these have 10x the thickness?

Reply:
Vector lines are in the context of the object they're set on, including scale.  Playmat is 10X so has to be 1/10th the width.

Reply:
Ah that makes sense